src/shell_minimal.html

```diff
@@ -116,7 +116,7 @@
           if (text === Module.setStatus.text) return;
           var m = text.match(/([^(]+)\((\d+(\.\d+)?)\/(\d+)\)/);
           var now = Date.now();
-          if (m && now - Date.now() < 30) return; // if this is a progress update, skip it if too soon
+          if (m && now - Module.setStatus.last < 30) return; // if this is a progress update, skip it if too soon
```

Comment:
Oh wow, this is a silly bug, thanks for noticing!

I'm not sure it's quite right yet, though - where is `Module.setStatus.last` set? Seems like it should be updated after this line. Also, the initial set higher up assigns an object there, so I think we need an extra `.time` to access the right property?

Also, we don't seem to use the `.text` field there - I think the idea was to use it to prevent writing the same text twice.

Might be worth thinking about how to test this. Although it's hard to test timing things without random errors...

Please also add yourself to AUTHORS.

Reply:
On closer inspection we need to check Module.setStatus.last_.time_ , I updated the patch (it used to update the status all the time).

There's a line to init Module.setStatus.last when it doesn't exist, but nothing to update it indeed, so I added that as you suggested.

Testing this is hard indeed, I had to add a large --preload-file to observe the result.

Gladly added myself to AUTHORS :)

I also changed the width of #progress from 30px to 300px (same as the default screen width) in shell.html, since 30px is really small - much smaller than in shell_minimal.html.